src/coredump/coredump.c

```diff
@@ -1115,18 +1115,43 @@ static int process_socket(int fd) {
                                 /* We have all FDs we need let's take a shortcut here. */
                                 break;
                         } else {
```

Comment:
get rid of the `else` here? the `if` branch does `break` anyway, hence you can get rid of one level of indentatin here by dropping the `else`.

We generally prefer shallow indentation

Reply:
Unaddressed.